### PR TITLE
Initial topo function

### PR DESCRIPTION
**Pull request by anne-glerum**
Add a function to describe initial topography.

### CODE REVIEW

File: tests/box_initial_topography_function.prm [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# This input file tests the box geometry in
+# combination with the ascii data initial
```
Comment (gassmoeller) [inline tests/box_initial_topography_function.prm]: ascii data -> function

File: tests/box_initial_topography_function.prm [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 660000
+    set Y extent = 660000
+    set X repetitions = 1
+    set Y repetitions = 1
+  end
+# We specify the name and directory of the ascii data file
```
Comment (gassmoeller) [inline tests/box_initial_topography_function.prm]: Remove comment.

File: source/geometry_model/initial_topography_model/function.cc [hunk 3]
```diff
@@ -0,0 +1,165 @@
+/*
+  Copyright (C) 2011 - 2017 by the authors of the ASPECT code.
```
Comment (gassmoeller) [inline source/geometry_model/initial_topography_model/function.cc]: 2019

**Comment by anne-glerum** [comment]
done @gassmoeller :)

**Pull request merged**